day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,22 @@
+# 10 Little Monkeys App
+# version 1.0
+# by Hashim Gari
+
+# uses integer parameter to loop through and print a nursury rhyme
+def print_story(num_monkeys)
```

Comment:
How does the rhyme change for the last monkey? How would you adjust your code accordingly?